src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Popup/Popup_Simple.xaml.cs

```diff
@@ -1,14 +1,53 @@
+using Windows.UI.Xaml;
 using Uno.UI.Samples.Controls;
 using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
 
 namespace Uno.UI.Samples.Content.UITests.Popup
 {
-	[SampleControlInfo("Popup", "Popup_Simple")]
+	[SampleControlInfo("Popup", "Popup_Simple", description: "Description for sample of Popup_Simple")]
```

Comment:
Description not implemented :p

Reply:
It's just a copy of what was in the XAML...  I should have read it instead of just moving it ;-)